### PR TITLE
Added a responsive memories section

### LINKED ISSUE #281: Make memories section responsive
**Actual Behaviour**

 memories section added isnt resposive. The image added should reduce in number on screens with less width. shrunken images look weird.

**Steps to reproduce it**
go to the [original website](https://gci18.fossasia.org)

**Would you like to work on the issue?**
Open to all

### PR DESCRIPTION
**Pull request by Sid911**
# Changes done in this Pull Request

- [Preview Link] https://sid911.github.io/gci18/
- Fixes #281 

## Description / Changes
Added a more responsive section instead of the old unresponsive one

### Before:
image

image

### After:
image

image

### CODE REVIEW

**Comment by Sid911** [comment]
I can't really figure out how to stop the scroll....

File: index.html [hunk 1]
```diff
@@ -211,38 +211,86 @@ <h1>Connect with Fossasia on Social Media</h1>
...
+    <a href="img/PastMemories/5.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/5.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/5.jpg" style="display: none;">
+    </div>
+    </a>
+    </li>
+    <li>
+    <a href="img/PastMemories/6.jpg" data-lightbox="Gallery">
+    <div class="background-image-holder fadeIn" style="background: url(&quot;img/PastMemories/6.jpg&quot;);">
+    <img alt="image" class="background-image" src="img/PastMemories/6.jpg" style="display: none;">
```
Comment (sakshee-19) [inline index.html]: Could you please also indent each of the tags properly, as it was before doing changes ?

**Comment by Sid911** [comment]
@sakshee-19  @fragm3 Done.... maybe 😊 depending if the "indenting the tags" meant formatting.

**Comment by Sid911** [comment]
I formatted the entire document using prettier and resolved problems like... extra tags.. wrong endings... no endings etc etc.

**Pull request merged**